docs/self-managed/components/orchestration-cluster/core-settings/concepts/elasticsearch-and-opensearch.md

```diff
@@ -0,0 +1,92 @@
+---
+id: elasticsearch-and-opensearch
+title: "Elasticsearch and Opensearch"
+description: "The Orchestration Cluster stores and reads data from either Elasticsearch or OpenSearch."
+---
+
+The Orchestration Cluster stores and reads data from either **Elasticsearch** or **OpenSearch**. You can select the database by setting the `CAMUNDA_DATABASE` environment variable or the equivalent configuration property.
+
+**Valid values:** `elasticsearch` (default) or `opensearch`.
+
+```bash
+CAMUNDA_DATABASE=opensearch
+```
+
+As of version 8.4, the cluster supports [Amazon OpenSearch](https://aws.amazon.com/de/opensearch-service/) 2.5.x. Using Amazon OpenSearch requires a new Camunda installation. Migration from previous Elasticsearch setups is not supported.
+
+## Connection settings
+
+To connect to a secured (HTTPS) Elasticsearch or OpenSearch instance:
+
+- **URL protocol**: Use `https` instead of `http`.
+- **Authentication**: Set a valid username/password combination for basic authentication.
+- **AWS credentials**: For Amazon OpenSearch, set `awsEnabled` to `true` to use AWS credentials instead of basic auth.
+- **SSL/TLS**: SSL settings such as `selfSigned` or disabling hostname verification should only be used if connection issues occur.
+- **JVM certificates**: Certificates may need to be imported into the JVM runtime.
+
+You can specify either `host` and `port` (deprecated) or `url` (recommended).
+
+## Common configuration options
+
+| Name             | Description                                  | Default value                                  |
+| ---------------- | -------------------------------------------- | ---------------------------------------------- |
+| cluster-name     | Name of the Elasticsearch/OpenSearch cluster | elasticsearch / opensearch                     |
+| url              | URL of the cluster REST API                  | [http://localhost:9200](http://localhost:9200) |
+| username         | Username to access the cluster REST API      | -                                              |
+| password         | Password to access the cluster REST API      | -                                              |
+| certificate-path | Path to SSL certificate                      | -                                              |
+| self-signed      | Certificate is self-signed                   | false                                          |
+| verify-hostname  | Validate hostname for SSL                    | false                                          |
+| awsEnabled       | Use AWS credentials (OpenSearch only)        | false                                          |
```

Comment:
Not sure what `awsEnabled` is supposed to be and confirmed if this is for 8.8? CC @buccarel